scripts/tests/run_java_test.py

```diff
@@ -63,7 +63,7 @@ def main(app: str, app_args: str, tool_path: str, tool_cluster: str, tool_args:
             retcode = subprocess.call("rm -f %s" % kvs_path_to_remove, shell=True)
             print("Trying to remove KVS path %s" % kvs_path_to_remove)
             if retcode != 0:
-                raise Exception("Failed to remove %s for factory reset." % kvs_path_to_remove)
+                raise OSError(f"Failed to remove {kvs_path_to_remove} for factory reset.")
```

Comment:
`OSError` is used when a syscall returns an error. I think `RuntimeError` is applicable here like in other places where extenal commands fail.

Reply:
Fixed